index.html

```diff
@@ -0,0 +1,106 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <main id="main">
+            
+            <h1 id="title"> Dr. Norman Borlaug</h1>
+            <p>The man who sved a billion lives</p>
+
+            <figure id="img-div">
+                <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
+           <figcaption id="img-caption">
+           <p id="txt">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+           </p>
```

Comment:
It is very clear and good coding standard, if you manage your codes in a same line with indentation.

Reply:
- [ ] Hi @abdullah97-pro 👋, Well done on using inline comments and giving a polite suggestion with a reason. 👍